### PR TITLE
Update CLI to use new `test` terminology and statuses

### PR DESCRIPTION
**Pull request by tmeasday**
For #286

### CODE REVIEW

File: bin/tasks/report.js [hunk 1]
```diff
@@ -61,30 +63,35 @@ export const generateReport = async (ctx) => {
...
-        break;
-      default:
+      default: {
+        switch (result) {
+          case 'SKIPPED':
+          case 'PRESERVED':
+            testCase.skipped();
+            break;
+          default:
+        }
```
Comment (ghengeveld) [inline bin/tasks/report.js:93]: Seems a bit overkill to use a switch for something that could just be a single if statement.

File: bin/tasks/report.js [hunk 2]
```diff
@@ -16,8 +16,9 @@ const ReportQuery = `
         cachedUrl
         createdAt
         completedAt
-        snapshots {
+        tests {
           status
+          result
```
Comment (ghengeveld) [inline bin/tasks/report.js:21]: Did you add `result` to `testFields` in `authorizedBuildFieldsViaAppCode`?

**Review by ghengeveld — CHANGES_REQUESTED** [review]

File: bin/tasks/report.js (hunk 2 above)
Comment (tmeasday) [reply bin/tasks/report.js:21]: Good point https://github.com/chromaui/chromatic/pull/4504

File: bin/tasks/report.js [hunk 3]
```diff
@@ -61,30 +63,31 @@ export const generateReport = async (ctx) => {
...
+  build.tests.forEach(({ status, result, spec, parameters }) => {
     const suffix = parameters.viewportIsDefault ? '' : ` [${parameters.viewport}px]`;
     const testCase = suite
       .testCase()
       .className(spec.component.name.replace(/[|/]/g, '.')) // transform story path to class path
       .name(`${spec.name}${suffix}`);
 
     switch (status) {
-      case 'SNAPSHOT_ERROR':
+      case 'FAILED':
```
Comment (ghengeveld) [inline bin/tasks/report.js:74]: You'll also need to change `status` to `status(legacy: false)` in the gql query above I think.

**Review by ghengeveld — CHANGES_REQUESTED** [review]

**Comment by tmeasday** [comment]
@ghengeveld I think this is ready now

**Comment by ghengeveld** [comment]
Released in [v5.7.0](https://github.com/chromaui/chromatic-cli/blob/next/CHANGELOG.md#570---2021-03-11).

**Pull request merged**